File: twiml/voice/client/client-2/client-2.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var dial = new Dial(callerId: "+1888XXXXXXX");
        dial.Number("858-987-6543");
        dial.Client("joey");
        dial.Client("charlie");
        response.Append(dial);

        Console.WriteLine(response.ToString());
    }
}

File: twiml/voice/client/client-2/client-2.5.x.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;


const response = new VoiceResponse();
const dial = response.dial({
    callerId: '+1888XXXXXXX'
});
dial.number('858-987-6543');
dial.client('joey');
dial.client('charlie');

console.log(response.toString());
